languages/ruby/exercises/concept/enumeration/.docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+Enumeration is the act of stepping through a collection (Array, Hash, etc) and performing some action on each object.
+Enumeration is a key concept in Ruby and is used for sorting (`sort_by`), grouping (`group_by`), mapping (`map`), reducing (`reduce`), and much more. You'll most frequently see enumeration as the idiomatic way iterating through collections rather than using loops.
+
+A simple enumeration to print each word in an array would look like this:
+
+```ruby
+words = %w[the cat sat on the mat]
+words.each do |word| 
+  print word 
+end
+
+#=> the
+#=> cat
+#=> ...
+#=> mat
+```
```

Comment:
I was slightly confused about the `#=> ...`, maybe use less words or list all the outputs?